lib/inferno/apps/web/serializers/test.rb

```diff
@@ -7,7 +7,7 @@ class Test < Serializer
         field :short_id
         field :title
         field :short_title
-        field :input_definitions, name: :inputs, extractor: HashValueExtractor, blueprint: Input
+        field :available_inputs, name: :inputs, extractor: HashValueExtractor, blueprint: Input
```

Comment:
Why the naming disparity between `available_inputs` and `output_definitions`?  Is that because you want to indicate that technically not all inputs will show up at the group level, in case they are being chained between tests or subgroups?

Reply:
The previous PR changed how inputs are handled, and I opted to call the class that defines an input `Input` rather than `InputDefinition`, and I changed the names of various methods to just refer to inputs rather than input_definitions to match this naming.

Similar changes weren't made to outputs, so they didn't get similar terminology changes. These are just the behind-the-scenes method names, not what is exposed via the API.

Reply:
The serializer isn't what defines what ends up in the API?  Or is it just serializing it to the DB?

Reply:
Nevermind, I just realized it is serializing it to `inputs`!